src/main.js

```diff
@@ -3,8 +3,17 @@ import {main} from './program';
 import cmd from './cmd';
 import * as logger from './util/logger';
 
-// This only exposes util/logger so far.
+// This only exposes util/logger and a subset of util/adb so far.
 // Do we need anything else?
-const util = {logger};
+const util = {
+  logger,
+  // Lazy load the adb module when util.adb is accessed for the first time, to defer loading
+  // the npm dependencies used by the adb module to when actually needed.
+  // This is being done to continue the courtesy of web-ext issue #1301
+  get adb(): Object {
+    const {listADBDevices, listADBFirefoxAPKs} = require('./util/adb.js');
```

Comment:
@ankushduacodes Nit (sorry I missed this one in my last review pass), I think we can omit `.js` from this `require`, right?

Reply:
Yes, it works. But does mentioning `.js` look more verbose here? as it indicates that it not a directory?

Reply:
> Yes, it works. But does mentioning `.js` look more verbose here? as it indicates that it not a directory?

well, we are not including the extension in the imports we have in the same file and so omitting it from the require would be good for consistency (the only case when we want to include the file extension explicitly is when it doesn't work without it, as in `tests/fixtures/webext-as-library/test-import.mjs` file in this patch)

we don't care if `util/adb` turns out to be a directory with an `index.js` file in it, on the contrary if we would ever decide to split the `adb` module into multiple files then we would need to change the `require` here is we don't want to.